pkg/middleware/oapi_validate.go

```diff
@@ -18,8 +20,18 @@ import (
 // Options to customize request validation, openapi3filter specified options will be passed through.
 type Options struct {
 	Options openapi3filter.Options
+	ErrRespContentType
 }
 
+type ErrRespContentType string
```

Comment:
Add a comment here for documentation purposes.
`// ErrRespContentType represents the content type for the response when validation errors occur...blah blah blah`